Database/connectMongoDb.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
```

Comment:
this should be present in the server.js file at the top so that the entire app will be able to use it.

Reply:
removed